storage/storage_image.go

```diff
@@ -341,8 +341,9 @@ func (s *storageImageSource) GetSignatures(ctx context.Context, instanceDigest *
 
 // newImageDestination sets us up to write a new image, caching blobs in a temporary directory until
 // it's time to Commit() the image
-func newImageDestination(imageRef storageReference) (*storageImageDestination, error) {
-	directory, err := ioutil.TempDir(tmpdir.TemporaryDirectoryForBigFiles(), "storage")
+// If SystemContext.BigFilesTemporaryDir not "", overrides the temporary directory to use for storing big files
```

Comment:
(Absolutely nonblocking: `BigFilesTemporaryDir` does not need to be documented here. But it does not hurt either.)